### PR TITLE
Open/Save Mapping fix

### PR DESCRIPTION
**Pull request by costebano**
- fixed openMapping;
- fixed saveMapping;

### CODE REVIEW

File: UI/src/app/components/comfy/comfy.component.ts [hunk 1]
```diff
@@ -241,14 +240,15 @@ export class ComfyComponent extends BaseComponent
...
+    const prefix = 'target';
 
     this.state.target.tables.map(table => {
-      if(this.COLUMNS_TO_EXCLUDE_FROM_TARGET.findIndex(name => name === table.name)<0)
-      {this.target[table.name] = {};
-      this.target[table.name].name = `${prefix}-${table.name}`;
-      this.target[table.name].first = table.name;
-      this.target[table.name].data = [table.name];}
+      if (this.COLUMNS_TO_EXCLUDE_FROM_TARGET.findIndex(name => name === table.name) < 0) {
+        this.target[table.name] = {};
```
Comment (etokareva) [inline UI/src/app/components/comfy/comfy.component.ts:247]: could we also refactor this part to :
`
this.target[table.name]={
name: `${prefix}-${table.name}`,
first: table.name,
data: [table.name]
}
` 
?
Also table.name could be moved into const

Thank you!

File: UI/src/app/components/comfy/comfy.component.ts [hunk 2]
```diff
@@ -262,42 +262,30 @@ export class ComfyComponent extends BaseComponent
...
-      } else {
-        copyArrayItem(
-          event.previousContainer.data,
-          event.container.data,
-          event.previousIndex,
-          event.container.data.length
-        );
-
-        const targetname = event.container.id.split("-")[1];
+      const container = event.container;
```
Comment (etokareva) [inline UI/src/app/components/comfy/comfy.component.ts:273]:
```ts

const {container , previousContainer, previousIndex} = event

```

File: UI/src/app/components/comfy/comfy.component.ts (hunk 1 above)
Comment (costebano) [reply UI/src/app/components/comfy/comfy.component.ts]: done

File: UI/src/app/components/comfy/comfy.component.ts (hunk 2 above)
Comment (costebano) [reply UI/src/app/components/comfy/comfy.component.ts]: done

**Pull request merged**